sdks/python/apache_beam/transforms/core.py

```diff
@@ -136,7 +140,7 @@ def set_element(self, windowed_value):
       self.windows = windowed_value.windows
 
 
-class DoFn(WithTypeHints, HasDisplayData):
+class DoFn(WithTypeHints, HasDisplayData, urns.RunnerApiFn):
```

Comment:
We are creating our own DoFn function and since that change has been merged into the master and released in the version 2.2.0, pylint start to complain that we are not implementing the function `to_runner_api_parameter`.

I'm not sure of the expected implementation and I cannot find any documentation about the RunnerApiFn. :( Can you point me to the right place for how to implement this method.

https://beam.apache.org/documentation/programming-guide/#pardo
https://beam.apache.org/documentation/sdks/pydoc/2.2.0/apache_beam.transforms.core.html#apache_beam.transforms.core.DoFn.to_runner_api_parameter
https://beam.apache.org/documentation/sdks/pydoc/2.2.0/apache_beam.utils.urns.html#apache_beam.utils.urns.RunnerApiFn

Reply:
urns.RunnerApiFn.register_pickle_urn  (below) should be providing an implementation of this for you.

Reply:
Hooo ok I see the implementation now, so I guess I'll just need to disable the pylint warning on all my classes that extends DoFn (or any other transform that now inherits from RunnerApiFn).
Except if you tell me that you have a better approach in mind?